tests/task/task_runner/test_standard_task_runner.py

```diff
@@ -131,38 +141,34 @@ def test_notifies_about_start_and_stop(self):
         )
         dag = dagbag.dags.get("test_example_bash_operator")
         task = dag.get_task("runme_1")
+        dag.create_dagrun(
+            run_id="test",
+            data_interval=(DEFAULT_DATE, DEFAULT_DATE),
+            state=State.RUNNING,
+            start_date=DEFAULT_DATE,
+        )
+        ti = TaskInstance(task=task, run_id="test")
+        job1 = LocalTaskJob(task_instance=ti, ignore_ti_state=True)
+        runner = StandardTaskRunner(job1)
+        runner.start()
+
+        # Wait until process sets its pgid to be equal to pid
```

Comment:
```suggestion
        # Wait until process makes itself the leader of it's own process group
```

Reply:
Though why do we have this loop, and then a 10s timeout directly below it?

Reply:
do not know, existing code :)

Reply:
looking, though, i think the timeout is related to "done" while the loop is about "started"